packages/config/config/devices/0x0002/keemple_smart_radiator.json

```diff
@@ -12,5 +16,21 @@
 	"firmwareVersion": {
 		"min": "0.0",
 		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "132",
+			"label": "Wake Up",
+			"description": "Wake Up Interval",
+			"valueSize": 60,
+			"minValue": 60,
+			"maxValue": 1800,
+			"defaultValue": 300
+		}
+	],
```

Comment:
This doesn't seem correct. 132 is the ID of the `Wake Up CC`, which has nothing to do with the configuration parameters that are defined through `paramInformation`.

Reply:
I've hoped that updating this command I can change Wake Up Interval in my battery thermostat.
I don't have any z-wave device that has parameters to configure, so I didn't realize I can't update Wake Up time it this way.
Deprecated OpenZWave integration had an option to set it through UI, do You know how can I change it in Z-Wave JS?

Reply:
See [my comment i](https://github.com/zwave-js/node-zwave-js/pull/3807#discussion_r759589266)